Sources/StreamChatUI/CommonViews/Attachments/AttachmentPreviewProvider.swift

```diff
@@ -49,3 +49,19 @@ extension VideoAttachmentPayload: AttachmentPreviewProvider {
         return preview
     }
 }
+
+extension VoiceRecordingAttachmentPayload: AttachmentPreviewProvider {
+    public static var preferredAxis: NSLayoutConstraint.Axis { .vertical }
```

Comment:
What's the point of placing this here? Clients will not be able to override it

Reply:
This is part of the `AttachmentPreviewProvider` protocol